Convex_hull_3/include/CGAL/convex_hull_3.h

```diff
@@ -1036,14 +1040,40 @@ void convex_hull_3(InputIterator first, InputIterator beyond,
 
 template <class InputIterator, class Polyhedron_3>
 void convex_hull_3(InputIterator first, InputIterator beyond,
-                   Polyhedron_3& polyhedron)
+                   Polyhedron_3& polyhedron,
+                   typename std::enable_if<
+                     CGAL::is_iterator<InputIterator>::value
+                   >::type* =0) //workaround to avoid ambiguity with next overload.
```

Comment:
You can use CGAL_BGL_NP_TEMPLATE_PARAMETERS / CGAL_BGL_NP_CLASS to avoid ambiguities due to named parameters.

Reply:
Thx, but it's not working for some reason, and I won't investigate further when I have already a working solution.